internal/services/containerapps/helpers/container_apps.go

```diff
@@ -2224,7 +2240,8 @@ func SecretsSchema() *pluginsdk.Schema {
 
 				"value": {
 					Type:        pluginsdk.TypeString,
-					Required:    true,
+					Optional:    true,
+					Computed:    true,
```

Comment:
Is there any reason to make this as `Computed: true`? Normally, we don't want to mark it as both `Optional` and `Computed`, and in case the API also returns `value` when you have `key_vault_secret_id` specified, we will require the users to explicitly mark the `value` in `ignore_change`.

Reply:
okay understood. the API does return `value` when `key_vault_secret_id` is specified. To support `ignore_change`, I'll need to update `SecretsSchema` to be `TypeList` right? as far as I'm aware, `TypeSet` can't be referenced in `ignore_change`

Reply:
Probably yes, I overlooked that the schema is a `TypeSet`. In order to make this change, you'll need to ensure the API returns secrets in a fixed order (i.e. the order as you specified in the request).

Reply:
great, that's what I thinking - will do some testing. Thanks for the clarification

Reply:
thankfully the API does return the secrets in the same order that they were created. I've updated to a list now, all seems to be working with `ignore_changes`. will update the docs shortly